src/models/IRoutePathLink.ts

```diff
@@ -6,7 +6,7 @@ export default interface IRoutePathLink {
     routePathDirection?: string;
     routePathStartDate?: Date;
     id: string;
-    positions: [[number, number]];
+    positions: [[number, number]]; // TODO: use geojson format instead (?)
```

Comment:
If it was geojson's geometry object, we could say:
```
<polyline
coordinates={geometry.coordinates}
</polyline
```

If we had other type of geometries than polygons, we would have geometry.type

At least that variable should be renamed as "coordinates" instead of positions

Reply:
Changed description of #367